packages/mobile/src/send/SendConfirmation.test.tsx

```diff
@@ -43,16 +43,17 @@ describe('SendConfirmation', () => {
       },
     })
 
-    const { toJSON, queryByText } = render(
+    const { toJSON } = render(
       <Provider store={store}>
         <SendConfirmation {...mockScreenProps} />
       </Provider>
     )
 
     // Initial render
     expect(toJSON()).toMatchSnapshot()
-    expect(queryByText('securityFee')).not.toBeNull()
-    expect(queryByText('0.001')).toBeNull()
+    // TODO: Find out why there is no handler function found for event "press"
```

Comment:
yeah the way testing-library react-native does on press seems to be to just call the prop rather than triggering an actual press event.